File: auto-docs/executables/node_js/box_plot_jitter.js
// Learn about API authentication here: {{BASE_URL}}/nodejs/getting-started
// Find your api_key here: {{BASE_URL}}/settings/api

var plotly = require('plotly')('TestBot', 'r1neazxo9w');
var data = [
  {
    y: [0, 1, 1, 2, 3, 5, 8, 13, 21], 
    boxpoints: "all", 
    jitter: 0.3, 
    pointpos: -1.8, 
    type: "box"
  }
];
var graphOptions = {filename: "box-plot-jitter", fileopt: "overwrite"};
plotly.plot(data, graphOptions, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/python/box_plot_jitter.py
# Learn about API authentication here: {{BASE_URL}}/python/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Box(
        y=[0, 1, 1, 2, 3, 5, 8, 13, 21],
        boxpoints='all',
        jitter=0.3,
        pointpos=-1.8
    )
])
plot_url = py.plot(data, filename='box-plot-jitter', auto_open=False)
